src/app.vue

```diff
@@ -34,5 +34,12 @@ export default {
       return this.store.inIframe ? "iFrameLayout" : "BaseLayout"
     },
   },
+  mounted() {
+    // migrate old situationId to simulationId
```

Comment:
Je n'ai pas assez de contexte pour comprendre ce que tu veux faire avec cette logique là.

Reply:
Je complète la description ⬆️

Reply:
Est ce que ça marche ? Est ce qu'on est sûr que ça fonctionne à tous les coups ? Pourquoi ne pas mettre ça directement dans le store ?

D'ailleurs faire un  `delete this.store.situationId` ça me parait pas très propre, en tant que manipulation du store sans passer par une fonction du store lui même.

Reply:
Ce code est fonctionnel. Voici les cas que j'ai envisagés et pour lesquels la logique est fonctionnelle :

 - [x] Store vide au lancement de l'application
 - [x] Store contenant une simulation précédemment terminée au lancement avec situationId non nul sur la page d'accueil
 - [x] Store contenant une simulation précédemment terminée au lancement avec situationId non nul sur la page de résultats
  
 `delete this.store.situationId` est fonctionnel. J'ai mis ces lignes dans une méthode dans le store à l'instant. Je ne l'ai pas fait à l'origine car ce n'était pas prévu pour être réutilisé ailleurs.